app/views/time_regs/_form.html.erb

```diff
@@ -63,7 +63,7 @@
                 <%= form.hidden_field :minutes, data: { "custom-input-target": "clone" } %>
                 <%= form.text_field :minutes_string, placeholder: "0:00", autocomplete: "off", data: {
                   action: "custom-input#cloneValue"
-                }, value: convert_time_int(time_reg.minutes),
+                }, value: (convert_time_int(time_reg.minutes) unless time_reg.new_record?),
```

Comment:
Just a question, I'm not sure I understand what this is adding considering `minutes` is 0 by default leading to `00:00`. I'm just trying to understand what this PR/Change is adding/changing @hoglund003

Reply:
When creating a new `TimeReg`, it is a bit annoying having to edit/overwrite the `0:00` value in the filed, rather than just inputting a new value and having the field blank.

Reply:
oh i see